Source/playerdat.cpp

```diff
@@ -317,9 +317,9 @@ const PlayerCombatData &GetPlayerCombatDataForClass(HeroClass clazz)
 	return PlayersCombatData[static_cast<size_t>(clazz)];
 }
 
-const PlayerStartingLoadoutData &GetPlayerStartingLoadoutForClass(HeroClass clazz)
+const PlayerStartingLoadoutData &GetPlayerStartingLoadoutForClass(HeroClass pClass)

```

Comment:
`playerClass` is what we use in `GetPlayerDataForClass` (also need to change the variable name for `GetPlayerCombatDataForClass` to match)